socorro/app/fetch_transform_save_app.py

```diff
@@ -36,6 +37,9 @@
 from socorro.app.socorro_app import App
 
 
+logger = logging.getLogger(__name__)
```

Comment:
Since the classes in here derive from `App` and that has a `self.logger`, we can use that logger rather than the module-level one here.